roles/ceph-rgw/tasks/main.yml

```diff
@@ -6,6 +6,10 @@
   include_tasks: pre_requisite.yml
   when: not containerized_deployment | bool
 
+- name: rgw pool creation tasks
```

Comment:
Why did you move the pools creation here ?

Shouldn't this be done after the multisite tasks ?

Reply:
@cbodley and I realized that pool creation should be happening before the rgw is starting up or any multisite tasks.